### PR TITLE
Helped Migrate to Appwrite Version 0.15.^ and Appwrite SDK 6.0.0

### PR DESCRIPTION
**Pull request by kekavc24**
Helped :
* Migrate the repo to use Appwrite version 0.15.^ and Appwrite SDK 6.0.0
* Fixed some but not all linter issues. I left the various unused variables. I assumed you had planned to use them in future.

### CODE REVIEW

**Comment by stnguyen90** [comment]
Thanks so much for your contribution! 🙏 Let me look into who can review this so we can get it merged in!

File: lib/providers/watchlist.dart [hunk 1]
```diff
@@ -19,6 +18,7 @@ import 'package:flutter/material.dart';
 
 class WatchListProvider extends ChangeNotifier {
   final String _collectionId = "watchlists";
+  static const String _bucketId = "Your bucket ID";
```
Comment (brandonroberts) [inline lib/providers/watchlist.dart]: The `_bucketId` is **"default"**

**Review by brandonroberts — CHANGES_REQUESTED** [review]
Overall LGTM, just needs the bucketId to be updated. Tested and working locally

File: lib/providers/entry.dart [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
 import 'package:netflix_clone/api/client.dart';
 import 'package:netflix_clone/data/entry.dart';
 import 'package:flutter/material.dart';
 
 class EntryProvider extends ChangeNotifier {
+  final Map<String, Uint8List> _imageCache = {};
 
-  Map<String, Uint8List> _imageCache = {};
+  static const String _collectionId = "movies";
+  static const String _bucketId = "Your bucket ID";
```
Comment (brandonroberts) [inline lib/providers/entry.dart]: Should be **"default"**

**Review by brandonroberts — APPROVED** [review]
LGTM

**Comment by brandonroberts** [comment]
Thanks @kekavc24! Nice work!

**Comment by kekavc24** [comment]
No problem 💯 Thought I'd help since maybe you guys are busy making Appwrite Awesome 😜😎

**Pull request merged**